source/NeutronaWand/Actions.h

```diff
@@ -605,6 +605,33 @@ void checkWandAction() {
                   WAND_BARREL_LED_COUNT = LEDS_48;
 
                   stopEffect(S_VOICE_BARREL_LED_5);
+                  //stopEffect(S_VOICE_BARREL_LED_29);
+                  stopEffect(S_VOICE_BARREL_LED_48);
+                  stopEffect(S_VOICE_BARREL_LED_60);
+                  playEffect(S_VOICE_BARREL_LED_48);
+
+                  wandSerialSend(W_BARREL_LEDS_48);
+
+                  /*
+                  i_num_barrel_leds = 29;
+                  WAND_BARREL_LED_COUNT = LEDS_29;
+
+                  stopEffect(S_VOICE_BARREL_LED_5);
+                  stopEffect(S_VOICE_BARREL_LED_29);
+                  stopEffect(S_VOICE_BARREL_LED_48);
+                  stopEffect(S_VOICE_BARREL_LED_60);
+                  playEffect(S_VOICE_BARREL_LED_29);
+
+                  wandSerialSend(W_BARREL_LEDS_29);
+                  */
+                break;
+
+                case LEDS_29:
```

Comment:
@nomakewan The option for 29 LED should not be added as a standard option. This was ONLY for me to use to debug and experiment with off-the-shelf components. The code is still needed to operate with that count, but it should not be allowed as a viable selection to end-users.